compiler/tflchef/tflite/src/Op/SVDF.cpp

```diff
@@ -25,9 +25,10 @@ void TFliteOpSVDF::filler(const tflite::Operator *op, TFliteImport *import,
                           tflchef::ModelRecipe *model_recipe) const
 {
   const std::vector<int32_t> &inputs = as_index_vector(op->inputs());
+  assert(inputs.size() == 5);
 
-  const bool hasBias = (inputs.size() == 5);
-  assert(inputs.size() == 4 || hasBias);
+  // optional input tensor idx has minus value.
+  const bool hasBias = (inputs.at(3) < 0);
```

Comment:
But in case `bias` is not exist index of input less than 0. Example:

https://github.com/Samsung/ONE/blob/7cfd6cd4c132efcf432d45e4e6275889229eeae1/compiler/tflchef/tflite/src/Op/FullyConnected.cpp#L32-L34